src/lib/sql/functions.sql.ts

```diff
@@ -85,6 +85,25 @@ select
   pg_get_function_result(f.oid) as return_type,
   nullif(rt.typrelid::int8, 0) as return_type_relation_id,
   f.proretset as is_set_returning_function,
+  case
+    when f.proretset and rt.typrelid != 0 and exists (
+      select 1 from pg_class c
+      where c.oid = rt.typrelid
+      -- exclude custom types relation from what is considered a set of table
+      and c.relkind in ('r', 'p', 'v', 'm', 'f')
+    ) then true
+  else false
+  end as returns_set_of_table,
+  case
+    when rt.typrelid != 0 then
+      (select relname from pg_class where oid = rt.typrelid)
+    else null
+  end as return_table_name,
+  case
+    when f.proretset then
+      coalesce(f.prorows, 0) > 1
+    else false
+  end as returns_multiple_rows,
```

Comment:
I'm thinking instead of `returns_multiple_rows` we can just return `prorows` as is (setting it as `null` if 0)

It's easy to confuse it with `is_set_returning_function` since they [mean the same thing](https://www.postgresql.org/docs/current/functions-srf.html#FUNCTIONS-SRF); rather what we're doing is adhering to PostgREST's idiosyncrasies